pkg/network/node/ovscontroller_test.go

```diff
@@ -802,6 +802,11 @@ func TestAlreadySetUp(t *testing.T) {
 			t.Fatalf("(%d) unexpected error from AddBridge: %v", i, err)
 		}
 		oc := NewOVSController(ovsif, 0, true, "172.17.0.4")
+		oc.vxlanPort = 4789
+		_, err := ovsif.AddPort("vxlan0", 1, "type=vxlan", `options:remote_ip="flow"`, `options:key="flow"`, fmt.Sprintf("options:dst_port=%d", 4789))
```

Comment:
maybe add a comment here explaining that we're not testing this part but we have to do this to make AlreadySetUp work